### ISSUE TITLE
About training detaills

### ISSUE THREAD

**Issue by jensenzhoujh** [researchmm/SiamDW#13]
Hi,

During training for SiamFC+, do you freeze the weights of first 7*7 conv as described in the paper? If yes, why I cannot find the corresponding operations in this code? And does the same operation applies for the SiamRPN+? 

Thanks for your time.

**Comment by JudasDie** [comment]
> Hi,
> 
> During training for SiamFC+, do you freeze the weights of first 7*7 conv as described in the paper? If yes, why I cannot find the corresponding operations in this code? And does the same operation applies for the SiamRPN+?
> 
> Thanks for your time.

There is no difference for results whether you freeze the last layer or not. And yes, I unfreeze the blocks gradually during training. For simplity, I provide the 'unfreezing all with smaller learning rate' version. You can try these two versions, similiar results come. RPN and FC use similiar training strategy.

**Comment by JudasDie** [comment]
> 
> 
> Hi,
> 
> During training for SiamFC+, do you freeze the weights of first 7*7 conv as described in the paper? If yes, why I cannot find the corresponding operations in this code? And does the same operation applies for the SiamRPN+?
> 
> Thanks for your time.

I provide a version with `freeze-out` strategy. See [backbone.py](https://github.com/researchmm/SiamDW/blob/master/lib/models/backbones.py).This issue will be closed.
